package.json

```diff
@@ -1,34 +1,37 @@
 {
-  "name": "graph-data-structure",
-  "version": "1.9.0",
-  "description": "A graph data structure with topological sort.",
-  "main": "index.js",
-  "files": [
-    "graph-data-structure.js"
-  ],
-  "scripts": {
-    "test": "mocha",
-    "prepublishOnly": "browserify index.js -s GraphDataStructure -o graph-data-structure.js"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/datavis-tech/graph-data-structure.git"
-  },
-  "keywords": [
-    "graph",
-    "data",
-    "structures",
-    "algorithms"
-  ],
-  "author": "Curran Kelleher",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/datavis-tech/graph-data-structure/issues"
-  },
-  "homepage": "https://github.com/datavis-tech/graph-data-structure#readme",
-  "devDependencies": {
-    "browserify": "^16.5.0",
-    "graph-diagrams": "^0.5.0",
-    "mocha": "^6.2.0"
-  }
+    "name": "graph-data-structure",
```

Comment:
An interesting change to indentation.

Makes me think maybe we could include a Prettier script in the project itself. https://github.com/datavis-tech/graph-data-structure/issues/29

Reply:
Yeah that was my VSCode prettier plugin. I usually keep a code linting, formatting and commit linting tool in my codebase.